agri-store-react/src/Components/Toast/Toast.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { ToastContainer, toast } from "react-toastify";
+
+import "react-toastify/dist/ReactToastify.css";
+
+function Toast({ type, mesg }) {
```

Comment:
suggestion: try to use full argument names like message or content instead of mesg.

Reply:
sure shivani will improve in next PR